### PR TITLE
Add tests.yml

### PR DESCRIPTION
**Pull request by gotanygrapes-lemonade**
- add tests.yml
- update README badge pending

### CODE REVIEW

**Comment by yang-l** [comment]
There are few tests failed, and need to be corrected before merging

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Lambda Deploy Buildkite Plugin
+# Lambda Deploy GitHub Actions
```
Comment (orien) [inline README.md:1]: This title doesn't look right.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 
-A [Buildkite plugin](https://buildkite.com/docs/agent/v3/plugins) for deploying AWS Lambda function code.
```
Comment (orien) [inline README.md:3]: Let's keep this introductory sentence.

File: README.md [hunk 3]
```diff
@@ -14,7 +14,6 @@ steps:
           path: path/to/deploy/code
           s3_bucket: deploybucket
           s3_key: deploy/key/name
-          region: ap-southeast-2
```
Comment (orien) [inline README.md:17]: It looks like the code uses `region`. I don't think we should remove this from the example.

If the linter is complaining, we'll need to add `region` as a parameter in the `plugin.yml` file.

**Review by orien — COMMENTED** [review]
Nice work.

**Review by yang-l — APPROVED** [review]

**Pull request merged**